packages/transformers/webmanifest/src/dependencies.js

```diff
@@ -0,0 +1,36 @@
+// @flow
```

Comment:
you could probably just inline this file into the transformer file. might be a bit simpler to read